front/app/components/PostShowComponents/Comments/CommentVote.tsx

```diff
@@ -261,80 +249,62 @@ const CommentVote = ({
     }
   };
 
-  if (!isNilOrError(comment)) {
-    const commentingVotingDisabledReason = get(
-      post,
-      'attributes.action_descriptor.comment_voting_idea.disabled_reason'
-    );
-    const isSignedIn = !isNilOrError(authUser);
-    const disabled =
-      postType === 'initiative'
-        ? !commentVotingPermissionInitiative?.enabled
-        : isSignedIn && commentingVotingDisabledReason === 'not_permitted';
-
-    if (!disabled || upvoteCount > 0) {
-      return (
-        <Container className={`vote ${className || ''}`}>
-          <UpvoteButton
-            onClick={handleVoteClick}
-            disabled={disabled}
-            className={`
+  const commentingVotingDisabledReason = get(
+    post,
+    'attributes.action_descriptor.comment_voting_idea.disabled_reason'
+  );
```

Comment:
Should we also get rid of this get?

Reply:
It's used 2x in this file.

Reply:
I think it's ok to get rid of it actually - just with an additional check for the property we want. I even caught an error in our types that went unnoticed.

Reply:
> I even caught an error in our types that went unnoticed.

Great. There's a good chance a few more will reveal themselves while we factor out `get`.